File: C++/Car/Car.cpp
#include <iostream>
#include <string>
using namespace std;
#include "Car.h"

// Constructor
Car::Car(string make, string model, string color, int year) {
    this->make = make;
    this->model = model;
    this->color = color;
    this->year = year;
}

// getter and setter methods
string Car::getMake() {
    return this->make;
}

void Car::setMake(string make) {
    this->make = make;
}

string Car::getModel() {
    return this->model;
}

void Car::setModel(string model) {
    this->model = model;
}

string Car::getColor() {
    return this->color;
}

void Car::setColor(string color) {
    this->color = color;
}

int Car::getYear() {
    return this->year;
}

void Car::setYear(int year) {
    this->year = year;
}

void Car::print() {
    cout << "Car Information:" << endl;
    cout << "Make: " + this->getMake() << endl;
    cout << "Model: " + this->getModel() << endl;
    cout << "Color: " + this->getColor() << endl;
    cout << "Year: " + to_string(this->getYear()) << endl;
}

File: Python/Car/Car.py

class Car:
    def __init__(self, make, model, color, year):
        self.make = make
        self.model = model
        self.color = color
        self.year = year
    
    def get_make(self):
        return self.make
    
    def set_make(self, make):
        self.make = make
        
    def get_model(self):
        return self.model
    
    def set_model(self, model):
        self.model = model
    
    def get_color(self):
        return self.color
    
    def set_color(self, color):
        self.color = color
        
    def get_year(self):
        return self.year
    
    def set_year(self, year):
        self.year = year
    
    def __str__(self):
        return f"This car is a {self.color} {self.make} {self.model} from {self.year}"
    
    def __repr__(self):
        return f"Car('{self.make}', '{self.model}', '{self.color}', '{self.year}')"
    
if __name__ == "__main__":
    car1 = Car("Chevrolet", "Corvette", "Blue", 2023)